tools/EMLassemblyline/eml2eal.xml

```diff
@@ -21,12 +21,12 @@
 		 <test expect_num_outputs="1"> 
             <param type="data" name="data" value="Assessing_the_importance_of_field_margins_for_bat_species.xml"/>
             <output_collection name="EML_templates" type="list">
-				<element file="annotations.txt" ftype="tabular"/>
-				<element file="keywords.txt" ftype="tabular" />
-				<element file="attributes_data_blary_&amp;_al.txt" ftype="tabular"/>
-				<element file="catvars_data_blary_&amp;_al.txt" ftype="tabular"/>
-				<element file="geographic_coverage.txt" ftype="tabular" />
-				<element file="personnel.tabular" ftype="tabular" />
+				<element name="annotations" file="annotations.txt" ftype="tabular"/>
+				<element name="keywords" file="keywords.txt" ftype="tabular" />
+				<element name="attributes_data_blary_&amp;_al" file="attributes_data_blary_&amp;_al.txt" ftype="tabular"/>

```

Comment:
You are controlling the R code so maybe we can avoid the & and other strange characters.

Reply:
The names of the outputs files are coming from the metadata file describing the data files... We can put rules on the R code to try look at all strange characters but it seems to me potentially better to let the original choice... You think we can't manae having & character on names here?

Reply:
Finally I modified the input XML metadata file to 'ot have these & characters